src/client.rs

```diff
@@ -65,10 +66,26 @@ impl ClientBuilder {
     /// Constructs a new `ClientBuilder`.
     ///
     /// This is the same as `Client::builder()`.
-    pub fn new() -> ClientBuilder {
-        ClientBuilder {
-            inner: async_impl::ClientBuilder::new(),
-            timeout: Timeout::default(),
+    pub fn new(need_proxy: bool) -> ClientBuilder {
```

Comment:
Changing this constructor would be a breaking change... and more frequently, the option would be a method of the builder.

I'd suggest putting the `get_proxies()` call in `build`, if the proxies haven't otherwise been configured. Whatcha think?

Reply:
Oh, I'm sorry for misunderstanding the meaning of `ClientBuilder options` (What I think is that adding an argument to ClientBuilder constructor).. It's so rediculous..
I have add two methods (`use_system_proxy` and `no_proxy`).  I hope it can match current `ClientBuilder` design.